docs/content/en/integration/openid-connect/seafile/index.md

```diff
@@ -92,10 +101,47 @@ identity_providers:
       userinfo_signed_response_alg: 'none'
```

Comment:
```suggestion
      token_endpoint_auth_method: 'client_secret_basic'
      userinfo_signed_response_alg: 'none'
```

Just making an educated guess at this one. I can help you confirm it if you'd like. Relevant docs/source:

- https://github.com/haiwen/seahub/blob/cbba3d918194e582342072a7931f93c86082a55d/seahub/oauth/views.py#L131-L135
- https://github.com/requests/requests-oauthlib/blob/v1.3.1/requests_oauthlib/oauth2_session.py#L285-L324

Reply:
[Authlib's docs](https://docs.authlib.org/en/latest/client/oauth2.html#client-authentication) say that both methods are possible, default being `client_secret_basic`; however, [POST might be buggy](https://github.com/requests/requests-oauthlib/issues/488). I can't find any trace of these options in Seafile's repo, so I guess it relies on default `client_secret_basic`.